lib/generators/activeadmin_addons/install/install_generator.rb

```diff
@@ -11,7 +11,8 @@ def setup_assets
         if use_webpacker?
           generate "activeadmin_addons:webpacker"
         else
-          generate "activeadmin_addons:assets"
+          puts "ActiveAdmin Addons requires ActiveAdmin installed with webpacker. "\
+          "Please run 'rails generate active_admin:install --webpacker' and then 'rails generate activeadmin_addons:install'"
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is too long. [125/100]